src/auth/integration-tests/README.md

```diff
@@ -11,6 +11,20 @@ env GOOGLE_CLOUD_PROJECT=rust-auth-testing \
   cargo test --features run-integration-tests -p auth-integration-tests
 ```
 
+### Workload Identity integration tests
+
+These tests requires more complex set up to run, like running from an Azure/AWS
```

Comment:
```suggestion
These tests requires a more complex setup to run, like running from an Azure/AWS
```

I was told "set up" is the verb, and "setup" the noun 🤷

Reply:
thanks for catching that up